config/bolt/config.yaml

```diff
@@ -454,4 +454,4 @@ compatibility:
     # Setting this value to false is deprecated.
     template_view: true
     # Set to 'false' to enable using a newer version of the setcontent parser.
-    setcontent_legacy: true
+    setcontent_legacy: false
```

Comment:
This can be removed. Bolt 4 will have no option for the "legacy". 

In fact, before beta, we'll need to thoroughly go through the config, and determine what needs to be culled.

Reply:
How is this old config used in the new version?

Reply:
I'd say the current version is about 95% copied from bolt 3, and needs some serious work. I wouldn't spend too much time on it in this PR, but we'll clean it up soon. Both for what Bolt 4 should be, and also to make it more "Symfony 4 app"-like. :-)